src/Gateway/Events/GuildStickersUpdate.php

```diff
@@ -12,7 +14,7 @@ class GuildStickersUpdate
     public string $guild_id;
 
     /**
-     * @var \Ragnarok\Fenrir\Parts\Sticker[]
+     * @var Sticker[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)